src/graph/graph.c

```diff
@@ -354,6 +397,19 @@ void Graph_ConnectNodes(Graph *g, EdgeDesc *connections, size_t connectionCount,
     }
 }
 
+size_t Graph_GetNodeLabels(const Graph *g, NodeID id, size_t **labels) {
```

Comment:
Accept a stack allocated array of type LabelID, array size should be graph's label count.
This is to avoid heap allocation.